src/lib/project-saver-hoc.jsx

```diff
@@ -44,34 +53,40 @@ const ProjectSaverHOC = function (WrappedComponent) {
             }
             // TODO: distinguish between creating new, remixing, and saving as a copy
             if (this.props.isCreatingNew && !prevProps.isCreatingNew) {
-                this.storeProject(null)
+                this.props.onShowCreatingAlert();
+                this.storeProject()
```

Comment:
Why did the null go away?

Reply:
Thanks for noticing -- maybe this was just a rebase quirk I didn't catch? (Without the null param, the param is undefined, which has been tested as having the same behavior; that's what we used to use)